library/ssl_cli.c

```diff
@@ -440,7 +440,8 @@ static void ssl_write_cid_ext( mbedtls_ssl_context *ssl,
     p = mbedtls_platform_put_uint16_be( p, ext_len );
 
     *p++ = (uint8_t) ssl->own_cid_len;
-    mbedtls_platform_memcpy( p, ssl->own_cid, ssl->own_cid_len );
+    /* Not using more secure mbedtls_platform_memcpy as cid is not that critical */
```

Comment:
It's more than "not that critical": it's public knowledge as it's transmitted in the clear over the wire.

Reply:
fixed